src/Tribe/Template.php

```diff
@@ -269,8 +278,8 @@ public function merge_context( $context = [], $file = null, $name = null ) {
 			$context = [];
 		}
 
-		// Applies local context on top of Global one
-		$context = wp_parse_args( (array) $context, $this->global );
+		// Applies new local context on top of Global + Previous local
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 70f3490c22fef321e9c61c2f123dafee</sub>